test/test_shared.cpp

```diff
@@ -89,6 +91,32 @@ using unit_test::TestContext;
 // check-testcase` (or one of its friends) from the command line.
 
 
+#ifdef _WIN32
+namespace {
+    // This does NOT work like on POSIX: The child will begin execution from the unit
+    // test entry point, not from where fork() took place.
+    DWORD winfork(std::string unit_test_name)
+    {
+        if (getenv("REALM_FORKED"))
+            return GetCurrentProcessId();
+
+        remove("winfork.bat");
+        std::ofstream myfile("winfork.bat");
```

Comment:
I don't think this will work when we have two unit tests running concurrently that are both forking because the file name is the same for each. Can we add the test name and some unique id to the file name? If you don't do this, you should use `NONCONCURRENT_TEST` in the unit tests that use `winfork()`